tools/archetype/binding.test/src/main/resources/archetype-resources/src/test/java/__bindingIdCamelCase__HandlerTest.java

```diff
@@ -60,6 +60,8 @@ public void setUp() {
 
     @Test
     public void initializeShouldCallTheCallback() {
+        // mock the function to prevent null exception.
+        when(thing.getConfiguration()).thenReturn(new demo1Configuration());
```

Comment:
Thanks - this indeed looks like it was forgotten in #4771 (and went unnoticed since...).